src/components/PropertyMap.tsx

```diff
@@ -445,6 +444,14 @@ const PropertyMap: FC<PropertyMapProps> = ({
         }}
         onLoad={(e) => {
           setMap(e.target);
+          const attributionButton = document.querySelector(
+            '.maplibregl-ctrl-attrib-button'
+          );
+          if (attributionButton) {
+            attributionButton.click();
```

Comment:
@DonovanAndrews300 TypeScript is throwing an error saying that `click` doesn't exist on `Element`